webdriver/newsession/create-1.py

```diff
@@ -0,0 +1,24 @@
+#META: timeout=long
```

Comment:
They have to be in separate files because of the timeout limit; even with a long timeout we have a limit of 60s. Starting a session means starting the browser which takes a few seconds, so we can't have many session-starting tests in the same file.